tests/population/test_add_new_birth_cohort.py

```diff
@@ -65,6 +64,50 @@ def test_FertilityCrudeBirthRate(base_config, base_plugins):
     # TODO: Write a more rigorous test.
     assert len(pop.age) > start_population_size, 'expect new simulants'
 
+    base_config.update({
+        'interpolation': {
+            'extrapolate': False
+        },
+        'time': {
+            'start': {'year': 2016},
+            'end': {'year': 2020},
+            'step_size': 30,
+        },
+    })
+
+    simulation = initialize_simulation(components, base_config, base_plugins)
+    simulation.data.write("covariate.live_births_by_sex.estimate",
+                          build_table(500, 1990, 2016, ('age', 'year', 'sex', 'mean_value')
+                                      ).query('age == 25').drop('age', 'columns'))
+    simulation.setup()
+    with pytest.raises(ValueError):
+        simulation.take_steps(6)
+
+    base_config.update({
```

Comment:
Either parameterize the test with configuration/data or write two separate tests for the different set ups.